docs/markdown/glossary.md

```diff
@@ -45,27 +44,27 @@ See instead: [Router](#router)
 
 ## Cyclic Transaction  
 
-A transaction where the destination account is the same account, on the same ledger, as the source account. This can be useful when rebalancing liquidity (to enable future payments), or when rebalancing stored value (to spread risk, or to take advantage of changing exchange rates).                                                                      
+A transaction where the destination account is the same account on the ledger as the source account. This can be useful when rebalancing liquidity (to enable future payments), or when rebalancing stored value (to spread risk, or to take advantage of changing exchange rates).                                                                      
 
 ## Destination Account 
 
 The account of the receiver whose address is included in the Interledger packet. Note that anyone can claim to have a certain Interledger address; address ownership is not enforced.                                                                       
 
 ## Destination Amount  
 
-The amount to be received by the receiver.                                                                      
+The amount to be received by the receiver through ILP.                                                                      
 
 ## Distributed Ledger
 
 A ledger that is operated by a group of entities and runs on multiple servers. This term is used somewhat interchangeably with "Blockchain" or "Decentralized Ledger".                                                                        
 
 ## Exchange Rate
 
-The price of one ledger's asset in terms of another ledger's asset. Routers may generate revenue from the difference in value between incoming and outgoing transfers. The exchange rate between source and destination is determined by the product of exchange rates at each hop.                                                                         
+The price of one ledger's asset in terms of another ledger's asset. Routers may generate revenue from the difference in value between incoming and outgoing transfers. The exchange rate between a sender and a receiver is determined by the product of exchange rates at each hop.                                                                         
 
 ## Fulfillment 
 
-A 32-byte value used to trigger the execution of a transfer. In most Interledger payments, the fulfillment is known only to the receiver (or in the case of the STREAM protocol it is known to the sender and the receiver).                                                                        
+Also known as the "preimage" or "hash preimage", fulfillment is  a 32-byte value used to trigger the execution of a transfer. In most Interledger payments, the fulfillment is known only to the receiver (or in the case of the STREAM protocol it is known to the sender and the receiver).                                                                        
```

Comment:
Missing an `a` before "fulfillment":

..."hash preimage", **a** fulfillment is...

Reply:
Fixed.